src/async/ComposeJob.vala

```diff
@@ -123,9 +123,9 @@ class ComposeJob : GLib.Object {
       call.add_param ("in_reply_to_status_id", this.reply_id.to_string ());
     } else if (this.quoted_tweet != null) {
       Cb.MiniTweet mt = quoted_tweet.retweeted_tweet ?? quoted_tweet.source_tweet;
-
-      this.text += " https://twitter.com/%s/status/%s".printf (mt.author.screen_name,
+      var quoted_url = "https://twitter.com/%s/status/%s".printf (mt.author.screen_name,
                                                                mt.id.to_string ());
```

Comment:
This line should now line up with the one above (3 spaces to the right if I'm not mistaken)